APYDataGridBundle.php

```diff
@@ -23,7 +23,6 @@ public function build(ContainerBuilder $container)
     {
         parent::build($container);
 
-        $container->addCompilerPass(new GridExtensionPass());
```

Comment:
I'm having an issue with the removal of this compilation pass : 

as a result, the `addColumnExtension` of the `Grid` class is no longer called during the compilation. Therefore the columns extensions are not loaded, resulting to an error of type : 

> No suitable Column Extension found for column type: number